### DISCUSSION TITLE
https://fccid.io/SK9AMI7

### DISCUSSION THREAD

**Discussion by xanderphillips** [bemasher/rtlamr#237] [Q&A]
My local electric company (Duke Energy of Indiana) uses this meter.  https://fccid.io/SK9AMI7
I see the AMI6 listed on the compatible meters list but no AMI7's. 
Passing no parameters I get no packets on my terminals.

Any suggestions greatly appreciated!

**Comment by User873902** [comment]
Also interested in this meter. Did you get it working?

**Reply by jgdpbains** [reply]
Any solution for this?
Have the same meter from BC Hydro.

**Comment by kb1ca** [comment]
BC Hydro uses an Openway system for the vast majority of their currently installed meters (https://www.bchydro.com/news/press_centre/news_releases/2011/itron-selected-smart-meters.html ). The old Itron meters used a remote reading protocol similar to R900. Openway, however, is completely and utterly incompatible. 

Look for a reference to 'Openway' somewhere on your meter face to confirm.
